editor/inserter/test/menu.js

```diff
@@ -64,7 +64,7 @@ const textEmbedBlock = {
 	category: 'embed',
 };
 
-describe( 'InserterMenu', () => {
+describe.skip( 'InserterMenu', () => {
```

Comment:
I did some heave debugging here and it looks like we are having the same kind of issues that are reported in Enzyme repository [here](https://github.com/airbnb/enzyme/issues/1163) and [here](https://github.com/airbnb/enzyme/issues/1153). 

In our case it might be a mix of those two. I tried to add recommended `wrapper.update();` call after the click event is simulated, but it didn't help. It's quite interesting scenario because the part of the component that depends on the component's state (https://github.com/WordPress/gutenberg/blob/ad316ab1f8cfe60aeeb45ff27d45336bfb214c3c/editor/inserter/menu.js#L384) gets properly updated, but the other part which also is conditionally rendered based on the components's state doesn't show up in the output of `wrapper.debug()` call. When I debugged it turned out that React component does everything properly behind the scenes. It looks like Enzyme gets out of sync.